AUTHORS.rst

```diff
@@ -0,0 +1,43 @@
+Authors
+=======
+
+The list of contributors in alphabetical order:
+
+- Achintya Rao
```

Comment:
:heart:

I'm used to my name appearing later in lists, as people usually go alphabetical by last name! Not suggesting you change this, though. ;)